### PR TITLE
Delegate olp 1157

### PR DESCRIPTION
**Pull request by kingpinXD**

### CODE REVIEW

File: action/init.go [hunk 1]
```diff
@@ -22,6 +22,9 @@ const (
 	UNSTAKE  Type = 0x12
 	WITHDRAW Type = 0x13
 
+	//network network_delegation
+	NETWORKDELEGATE Type = 0x04
```
Comment (kerhbal) [inline action/init.go:26]: do u want to name it as `NETWORK_DELEGATE` for better readability? and maybe we can use 0x5x... for all the network delegation

Comment (kingpinXD) [reply action/init.go:26]: Yeah , I think ill change it to ADD_NETWORK_DELAGATION , 
the others can be WITHDRAW_NETWORK_DELEGATION and REMOVE_NETWORK_DELEGATION

File: action/network_delegation/add_network_delegation.go [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+}
+
+func (n AddNetworkDelegation) Tags() kv.Pairs {
+	tags := make([]kv.Pair, 0)
+
+	tag := kv.Pair{
+		Key:   []byte("tx.type"),
+		Value: []byte(n.Type().String()),
+	}
+	tag3 := kv.Pair{
```
Comment (kerhbal) [inline action/network_delegation/add_network_delegation.go:37]: I think u mean tag2

File: action/init.go [hunk 3]
```diff
@@ -71,7 +74,7 @@ func init() {
 
 	serialize.RegisterInterface(new(Msg))
 	logger = log.NewLoggerWithPrefix(os.Stdout, "action")
-	txTypeMap  = TxTypeMap{}
+	txTypeMap = TxTypeMap{}
 	RegisterTxType(SEND, "SEND")
 	RegisterTxType(SENDPOOL, "SENDPOOL")
```
Comment (kerhbal) [inline action/init.go:80]: u probably want to register tx type here

**Review by kerhbal — APPROVED** [review]

**Pull request merged**